evap/student/tests/test_views.py

```diff
@@ -351,6 +353,18 @@ def test_answer(self):
         ).values_list("answer", flat=True)
         self.assertEqual(list(answers), ["some bottom text"] * 2)
 
+    def test_vote_timestamp(self):
+        time_before = datetime.datetime.now()
+        timestamps_before = len(VoteTimestamp.objects.all())
+        page = self.app.get(self.url, user=self.voting_user1, status=200)
+        form = page.forms["student-vote-form"]
+        self.fill_form(form)
+        form.submit()
+        self.assertEqual(len(VoteTimestamp.objects.all()), timestamps_before + 1)
```

Comment:
```suggestion
        self.assertEqual(VoteTimestamp.objects.count(), timestamps_before + 1)
```
doesn't actually fetch them all (`select * from table_name`) but queries the count from the database (`select count(*) from table_name`).